src/components/nodes-table/nodes-table.js

```diff
@@ -273,10 +273,16 @@ export default {
 				description = 'mains-powered'
 			} else {
 				label = `${level}%`
-				description = Array.isArray(node.batteryLevels)
-					? 'All battery levels: ' +
-					  node.batteryLevels.map((v) => `${v}%`).join(',')
-					: 'Unknown battery level'
+				description =
```

Comment:
Same as above, also giving that this is the same function consider moving it to `lib/utils.js` and create a function `getBatteryDescription(node)`